tests/sentry/integrations/utils/test_code_mapping.py

```diff
@@ -271,64 +271,6 @@ def test_list_file_matches_multiple(self):
         ]
         assert matches == expected_matches
 
-    def test_normalized_stack_and_source_roots_starts_with_period_slash(self):
```

Comment:
These tests were for a function that was required for the previous implementation of root derivaiton. This function is no longer used, thus we don't need its tests.

Reply:
Can we keep them an use `find_roots` instead?